### ISSUE TITLE
Gujarati Translation Progress

### ISSUE THREAD

**Issue by tesseralis** [reactjs/gu.reactjs.org#1]
## Maintainer List

* @3sanket3
* @ron4ex
## For New Translators

To translate a page:

1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the name of the page you would like to translate. **Please take only one page at a time**.
3. Clone this repo, translate your page, and submit a pull request!

Before contributing, read the glossary and style guide (once they exist) to understand how to translate various technical and React-specific terms.

Please be prompt with your translations! If you find find that you can't commit any more, let the maintainers know so they can assign the page to someone else.

## For Maintainers

When someone volunteers, edit this issue with the username of the volunteer, and with the PR. Ex:

* [ ] Home Page (@tesseralis) #1

When PRs are merged, make sure to mark that page as completed!

## Core Pages

To do before releasing as an "official" translation. **Please translate these pages first.**

* [x] Home Page (@asifvora) #2 
* [ ] Tutorial

### Basics

* [x] Hello World (@hardikpthv) #3
* [x] Introducing JSX (@hardikpthv) #4
* [x] Rendering Elements (@sonipreshita) #5
* [ ] Components and Props (@Sunil-Makwana)
* [ ] State and Lifecycle (@harshcrop)
* [ ] Handling Events (@Rohitguptab)
* [ ] Conditional Rendering (@mansitrivedi26)
* [ ] Lists and Keys (@sadikmultani)
* [ ] Forms (@kuldipem)
* [ ] Lifting State Up (@kuldipem)
* [ ] Composition vs Inheritance (@iamrmin)
* [ ] Thinking in React

### API Reference

* [ ] React (@kamal250)
* [ ] React.Component
* [ ] ReactDOM
* [ ] ReactDOMServer (@pixan198)
* [ ] DOM Elements
* [ ] SyntheticEvent
* [ ] Test Utilities
* [ ] Shallow Renderer (@amincharoliya)
* [ ] Test Renderer
* [x] JS Environment Requirements (@Rutulpatel7077) #12 
* [ ] Glossary

### Navigation

These are the navigation links that appears in the sidebar. Possibly wait until
the corresponding sections are translated to do these.

* [ ] tutorial/nav.yml
* [ ] docs/nav.yml

## Next Steps

These are the next most important translations, ordered by importance:

### Installation

* [ ] Getting Started (@vieee)
* [ ] Add React to a Website (@arnavs-0)
* [ ] Create a New React App (@devsapariya94)
* [x] CDN Links (@PhoenixCreation)

### Hooks

Everyone is hooked on hooks! People are really excited for these and it would be great for them to be translated.

  * [ ] Introducing Hooks (@harshilparmar)
  * [ ] Hooks at a Glance (@aachauhan2009)
  * [ ] Using the State Hook (@devsapariya94)
  * [ ] Using the Effect Hook
  * [ ] Rules of Hooks (@csudani7)
  * [ ] Building Your Own Hooks
  * [ ] Hooks API Reference
  * [ ] Hooks FAQ

### Warnings

These are the pages that you get when you click the links in the console (e.g. https://reactjs.org/warnings/dont-call-proptypes.html). People tend to search these, so **please don't translate the error message itself**.
 
* [ ] dont-call-proptypes
* [ ] invalid-aria-prop
* [ ] invalid-hook-call-warning
* [ ] legacy-factories
* [ ] refs-must-have-owner
* [ ] special-props
* [ ] unknown-prop

### Advanced Guides

* [ ] Accessibility
* [ ] Code-Splitting
* [ ] Context
* [ ] Error Boundaries
* [ ] Forwarding Refs
* [ ] Fragments
* [ ] Higher-Order Components
* [ ] Integrating with Other Libraries
* [ ] JSX In Depth
* [ ] Optimizing Performance
* [ ] Portals
* [ ] React Without ES6
* [ ] React Without JSX
* [ ] Reconciliation
* [ ] Refs and the DOM
* [ ] Render Props
* [ ] Static Type Checking
* [ ] Strict Mode
* [ ] Typechecking With PropTypes
* [ ] Uncontrolled Components
* [ ] Web Components

### FAQ

* [ ] APIs and AJAX
* [ ] Babel, JSX, and Build Steps
* [ ] Passing Functions to Components
* [ ] Component State
* [ ] Styling and CSS
* [ ] File Structure
* [ ] Versioning Policy
* [ ] Virtual DOM and Internals

### Contributing

Priority: Low

* [ ] How to Contribute
* [ ] Codebase Overview
* [ ] Implementation Notes
* [ ] Design Principles

### Components

Components in `src/components` that have some text in them.

* [ ] CodeEditor
* [ ] CodeExample
* [ ] ErrorDecoder
* [ ] LayoutFooter
* [ ] LayoutHeader
* [ ] MarkdownPage

## Additional Translations

These are not the primary translation targets.

* [ ] Blog
* [ ] Community

**Comment by tesseralis** [comment]
@3sanket3 @ron4ex please use this issue to keep track of your progress. Let me know if I can be of any help!

**Comment by ron4ex** [comment]
Got it @tesseralis. I'll be taking up Tutorial page to begin with.

**Comment by asifvora** [comment]
Hello, @tesseralis @ron4ex I have been translated the home page and apply to the 1st pull request.

**Comment by ron4ex** [comment]
Hey @asifvora, thank you for submitting the pull request. If you're working on a translation, just let us know in the comments here before you begin your work. So your efforts don't go to waste due to duplications.

I'm reviewing your PR right now and it seems to be a direct translation using _Google Translate_ of the English version. That is not what we're looking to do here. And as you can see, the result is not a reliable translation.

**Comment by asifvora** [comment]
Hey, @ron4ex thanks for quick reply.
Yes, I will like to take another dig at the Home page translation.

**Comment by hardikpthv** [comment]
@tesseralis, @3sanket3 and @ron4ex, I'll be picking up Hello World Page to get started.

**Comment by hardikpthv** [comment]
@tesseralis, @3sanket3 and @ron4ex, Taking up next i.e. Introducing JSX Page.

**Comment by sonipreshita** [comment]
@tesseralis, @3sanket3 and @ron4ex, I am taking next **Rendering Elements**

**Comment by asifvora** [comment]
Hellow, @ron4ex 
I had done again Home page translation #2 .  Please check and review.

**Comment by Rutulpatel7077** [comment]
@tesseralis  I am in

**Comment by 3sanket3** [comment]
Hi @Rutulpatel7077, Thanks for your interest. You can start translating one of the page listed above in this issue(just update here which one you are taking). And, let me know if any info needed.

**Comment by Sunil-Makwana** [comment]
@ron4ex , I am in,  I'll be picking up "Components and Props" For translation.

**Comment by ron4ex** [comment]
@Sunil-Makwana Awesome. Checkout #6 (Glossary) if you have any doubts.

**Comment by Sunil-Makwana** [comment]
okay

**Comment by iamrmin** [comment]
@tesseralis Count me in! I am picking up **"Composition vs Inheritance"**

**Comment by ron4ex** [comment]
@iamrmin Great. Checkout #6 (Glossary) if you have any doubts regarding translation. Let me know if there's anything else.

**Comment by amincharoliya** [comment]
Hello @ron4ex ,

I'm interested to work on translation. Just to make sure everything goes right on the first hit I'm taking small page: **"Shallow Renderer"** under API Reference. Let me know should I start or not?

**Comment by ron4ex** [comment]
@amincharoliya Sure, go ahead. I've assigned it to you. Let me know if you have any questions. Also, don't forget to checkout #6 (Glossary)

**Comment by Rutulpatel7077** [comment]
@3sanket3 I am taking **JS Environment Requirements**. Please update (taken).

**Comment by ron4ex** [comment]
@Rutulpatel7077 Great. Updated.

**Comment by Rohitguptab** [comment]
Hello @ron4ex , @3sanket3 

I'm taking "Handling Events" page.

**Comment by mansitrivedi26** [comment]
Hello @ron4ex , @3sanket3 

I'm taking "Conditional Rendering" page.

**Comment by 3sanket3** [comment]
@Rohitguptab, @mansitrivedi26 Sure, go ahead. I have updated the issue description.

**Comment by mayur4856** [comment]
Hello @ron4ex , @3sanket3

I'm taking "Composition vs Inheritance" page.

**Comment by ron4ex** [comment]
@mayur4856 As you can see in the issue description, somebody is already working on that. See if you want to work on some other translation and let me know.

**Comment by sadikmultani** [comment]
Hello @ron4ex , @3sanket3

I'm taking "Lists and Keys" page.

**Comment by 3sanket3** [comment]
Sure @sadikmultani , assigned to you.

**Comment by mayur4856** [comment]
@ron4ex ,

Ok fine, i will work on another page.

**Comment by harshcrop** [comment]
can I start working on **State and Lifecycle** page? I want to take this page for start.

**Comment by ron4ex** [comment]
Sure @harshcrop, have a go :) Let me know if anything.

**Comment by harshcrop** [comment]
Thank you @ron4ex

**Comment by kuldipem** [comment]
@tesseralis @ron4ex @3sanket3 , I will work on **Forms**

**Comment by ron4ex** [comment]
Great @kuldipem, I've assigned it to you.

**Comment by kuldipem** [comment]
@tesseralis @ron4ex @3sanket3  ,  I will work on  **Lifting State Up**

**Comment by 3sanket3** [comment]
@kuldipem Sure assigning it to you.

**Comment by kuldipem** [comment]
@3sanket3, I've done my assigned tasks and also created related PRs, please review the translations, If you found any issue, please let me know, so I can take care of it.

**Comment by kuldipem** [comment]
In my free time, I've started work on **Installation** guide,  there related 4 page is almost ready. I will create PR soon.

**Comment by harshilparmar** [comment]
Can I start working on hooks page

**Comment by ron4ex** [comment]
Sure @harshilparmar . Assigned "Introducing Hooks"

**Comment by kamal250** [comment]
@ron4ex, I want to update  `API Reference -> React`.

**Comment by ron4ex** [comment]
> @ron4ex, I want to update `API Reference -> React`.

Assigned.

**Comment by PhoenixCreation** [comment]
@ron4ex Can I translate "Intallation => CDN"? I am native Gujarati.

**Comment by PhoenixCreation** [comment]
@ron4ex Is "**Installation**" is available? If not suggest anything that needs priority...

**Comment by ron4ex** [comment]
@PhoenixCreation Sure, anything that is unticked can be picked up. Go for it.

**Comment by PhoenixCreation** [comment]
@ron4ex Thanks. I am going for **Installation => CDN Links ** for now. Mark that one for me.

**Comment by PhoenixCreation** [comment]
I am going for **Installation -> create a new react-app** . This might take some time due to my exams but I'll surely finish it.

**Comment by aachauhan2009** [comment]
I want to pick up  Hooks at a Glance

**Comment by pixan198** [comment]
hey @3sanket3, @ron4ex 
I want to contribute for **API Reference/ReactDOMServer**

**Comment by vieee** [comment]
hi @3sanket3, @ron4ex
I want to contribute for `Getting Started` page in the `Installation` section..

**Comment by arnavs-0** [comment]
Hi @3sanket3 @ron4ex 
I'd like to work on **Add React to a Website**

**Comment by henilbhatt** [comment]
Hi @3sanket3 @ron4ex
I want to contribute for Thinking in React page.

**Comment by devsapariya94** [comment]
hey,
I want to translate Add React to a Website page

**Comment by 3sanket3** [comment]
Sorry for being late here. Can you all please confirm, so that I will update?

@arnavs-0 Are you still working on *Add React to a Website*

@henilbhatt Are you working on *Thinking in React*?

@devsapariya94: @arnavs-0 has already claimed to work on it, meanwhile we get confirmation from him, can you please take any other page?

**Comment by arnavs-0** [comment]
@3sanket3 Yes I am working on *Add React to a Website*, was just waiting on confirmation 😄

**Comment by 3sanket3** [comment]
Oh, yes please go ahead. Adding your name to the issue description.

@devsapariya94: Can you pick up another page, please?

**Comment by devsapariya94** [comment]
ok no problem i will work on " Create a New React App"

**Comment by devsapariya94** [comment]
I had translate Create a New React App, please Review it.
https://deploy-preview-128--gu-reactjs.netlify.app/docs/create-a-new-react-app.html

**Comment by imjimil** [comment]
Hello, I've Translated "invalid-hook-call-warning". 
review PR #132

**Comment by csudani7** [comment]
Hello @3sanket3 

I want to translate "**Using the State Hook**". can I go ahead?

**Comment by csudani7** [comment]
> Hello @3sanket3
> 
> I want to translate "**Rules of Hooks**". can I go ahead?

**Any Update guys?**

**Comment by 3sanket3** [comment]
> > Hello @3sanket3
> > I want to translate "**Rules of Hooks**". can I go ahead?
> 
> **Any Update guys?**

Please go ahead

**Comment by 3sanket3** [comment]
> hey,
> @3sanket3 I had translated "Create a New React App"
> please review PR #128
> and I want to translate "Using the State Hook"

@devsapariya94 Assigned to you, please go ahead

**Comment by chavda-bhavik** [comment]
Hey, @3sanket3 I would like to translate `Thinking in React`. Can you assign it to me?

**Comment by Krunalio** [comment]
Hey, @3sanket3 I would like to translate Thinking in React. Can you assign it to me?

**Comment by gaearon** [comment]
It looks like this translation has stalled, and is now very outdated.
I'm going to archive this repo, but if someone wants to pick up, please start a new effort.
